src/views/simplified-account/settings/card-payments/index.njk

```diff
@@ -1,10 +1,27 @@
 {% extends "../settings-layout.njk" %}
 
+{% from "govuk/components/notification-banner/macro.njk" import govukNotificationBanner %}
+
 {% block settingsPageTitle %}
   Card payments
 {% endblock %}
 
 {% block settingsContent %}
+  {% if not userCanUpdatePaymentTypes %}
+    {% include "simplified-account/settings/partials/read-only-message.njk" %}
+  {% endif %}
+  {% if flash.update %}
+    {% set html %}
+      <h3 class="govuk-notification-banner__heading">
+        {{ flash.update }}
+      </h3>
+    {% endset %}
+
+    {{ govukNotificationBanner({
+      html: html,
+      type: "success"
+    }) }}
+  {% endif %}
```

Comment:
this can all be removed, the `settings-layout` handles displaying messages (provided your controller sets flash appropriately, see previous comments)

Reply:
We've removed flash messages.